pkg/clusters/addons/kong/addon.go

```diff
@@ -372,6 +376,14 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 	args = append(args, "--namespace", a.namespace)
 	args = append(args, a.deployArgs...)
 	args = append(args, defaults()...)
+
+	if a.httpNodePort > 0 {
+		args = append(args, "--set", fmt.Sprintf("proxy.http.nodePort=%d", a.httpNodePort))
+	}
+	if a.adminNodePort > 0 {
+		args = append(args, "--set", fmt.Sprintf("admin.http.nodePort=%d", a.adminNodePort))
+	}
+
```

Comment:
:+1: 

(on a side note, it's been a while since I have looked at this code, I forgot how big these Deploy methods got! Not something we need to solve here though)